src/components/companies/index.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchCompanies } from '../../redux/companies/companySlice';
+import Company from './Company';
+import './index.css';
+
+const Companies = () => {
+  const dispatch = useDispatch();
+
+  const { companies, queryResults, isLoading } = useSelector(
+    (state) => state.companies,
+  );
+
+  useEffect(() => {
+    if (companies?.length !== 0) return;
+    dispatch(fetchCompanies());
+  }, [dispatch, companies?.length]);
+
+  if (isLoading) return <div className="loading">Loading...</div>;
+
+  if (queryResults?.length === 0) { return <div className="loading">No results found!</div>; }
+
+  return (
+    <main>
+      {queryResults.map((company, i) => (
+        <Company
+          key={company.symbol.concat(i)}
+          company={company}
+        />
```

Comment:
- [ ] As requested by the previous reviewer to follow the design guidelines, Please fix the below design issues.
  - Please make sure to display two items(companies) in a row. Currently, It's displaying only one item.
  - Please make sure to display a small button on each item, That will indicate users going to the details page.

### Your design

![image](https://github.com/microhass/thee-worth/assets/83803414/d123b679-275d-4740-9bae-0e1c2d58c4cf)

### Given design

![image](https://github.com/microhass/thee-worth/assets/83803414/33d636b1-9c33-48d5-8123-cc08978d5032)